### PR TITLE
Add Drupal

### PR DESCRIPTION
**Pull request by nirgn975**
**[https://github.com/nirgn975/awesome-drupal](https://github.com/nirgn975/awesome-drupal)**

**A collection of awesome Drupal resources to learn Drupal and it's modules.**

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Generally looks pretty good.

Some nitpick:

- Drop all the `A` / `An` prefixes in the descriptions.
- There should be a succinct summary at the top about Drupal.
- The `Data Migration` items should be reversed.
- Drop the Travis badge. It doesn't have any purpose for non-software repos.

File: readme.md [hunk 1]
```diff
@@ -375,6 +375,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Refinery CMS](https://github.com/refinerycms-contrib/awesome-refinerycms)
 - [Wagtail](https://github.com/springload/awesome-wagtail) – Django CMS focused on flexibility and user experience.
 - [Textpattern](https://github.com/drmonkeyninja/awesome-textpattern) - Lightweight PHP-based CMS.
+- [Drupal](https://github.com/nirgn975/awesome-drupal) - A free and open-source CMS written in PHP and distributed under the GNU License.
```
Comment (sindresorhus) [inline readme.md]: Would like to see something short, like:

> Extensible PHP-based CMS.

**Comment by nirgn975** [comment]
@sindresorhus 
Thanks for the feedback. Fix all of the above, and ready for review :)

**Comment by sindresorhus** [comment]
Looks good :)

**Pull request merged**